js/lib/beautify.js

```diff
@@ -279,6 +300,10 @@
         opt.e4x = (options.e4x === undefined) ? false : options.e4x;
         opt.end_with_newline = (options.end_with_newline === undefined) ? false : options.end_with_newline;
         opt.comma_first = (options.comma_first === undefined) ? false : options.comma_first;
+        opt.operator_position = options.operator_position;
+        
+        // throws error if invalid value was passed in
+        verifyOperatorPosition(opt.operator_position);
```

Comment:
Can you clarify "replacing undefined with the default"?  I'm expecting undefined to be the default value.

And will do.